### DISCUSSION TITLE
0.0.2308.061

### DISCUSSION THREAD

**Discussion by prosthetichead** [prosthetichead/GarlicPress#57] [Announcements]
## What's Changed
* Major rework of how game art is processed.
* New game art download screen for bulk processing art.
* Art is now processed Asynchronous so no more locking up
* Every Drive and Sytems game art can be updated in one job. (button can be found under tools)
* Can now cancel Art processing. To resume use clear completed button before starting again.
* Delete key deletes items when list is focused.
* Deleting a file now selects the next file not the top 
* Warning before delete can be disable. 
* Support for Alt Arcade systems (Final Burn 2012, Final Burn Neo, Mame 2000)
* Files can now be renamed
* Backup saves button is now moved under tools

<hr /><em>This discussion was created from the release <a href='https://github.com/prosthetichead/GarlicPress/releases/tag/0.0.2308.061'>0.0.2308.061</a>.</em>

**Comment by dmnthe** [comment]
Can you make it to support different (unknown) ROMS folders?

I downloaded the "tiny best set go extra unofficial" (from archive org) and it makes new folders of arcade games eg Taito, Toaplan, Visco and some of them are not in the list in Garlic Press so can't update their artwork
